daemon/main.go

```diff
@@ -110,6 +111,13 @@ var RootCmd = &cobra.Command{
 }
 
 func main() {
+
+	// Open socket for using gops to get stacktraces of the agent.
+	if err := gops.Listen(gops.Options{}); err != nil {
+		fmt.Println(err)
```

Comment:
What error message will this print? Will it be clear that this is gops related?

Reply:
I will add a custom error message like the following:
`fmt.Printf("unable to start gops: %s\n", err)`

Reply:
Fixed.